[01/Feb/2018 13:20:06] code 400, message Bad request syntax ('\x16\x03\x01\x00\xce\x01\x00\x00\xca\x03\x03\x9fB\xbb+\xe1._ydo\x9cp\x1a\xd78\x1c\x13\xec/8.\x16o\xea\x99\xefo\xe5gM\xc3Y\x00\x00\x1c\x9a\x9a\xc0+\xc0/\xc0,\xc00\xcc\xa9\xcc\xa8\xc0\x13\xc0\x14\x00\x9c\x00\x9d\x00/\x005\x00')
[01/Feb/2018 13:20:07] code 400, message Bad HTTP/0.9 request type ('\x16\x03\x01\x00\xce\x01\x00\x00\xca\x03\x03\x1a\x10J\xd5\xde')
Traceback (most recent call last):
  File "/usr/lib64/python2.7/SocketServer.py", line 593, in process_request_thread
Traceback (most recent call last):
  File "/usr/lib64/python2.7/SocketServer.py", line 593, in process_request_thread
    self.finish_request(request, client_address)
    self.finish_request(request, client_address)
  File "/usr/lib64/python2.7/SocketServer.py", line 334, in finish_request
  File "/usr/lib64/python2.7/SocketServer.py", line 334, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib/python2.7/site-packages/django/core/servers/basehttp.py", line 150, in __init__
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib/python2.7/site-packages/django/core/servers/basehttp.py", line 150, in __init__
    super(WSGIRequestHandler, self).__init__(*args, **kwargs)
    super(WSGIRequestHandler, self).__init__(*args, **kwargs)
  File "/usr/lib64/python2.7/SocketServer.py", line 649, in __init__
  File "/usr/lib64/python2.7/SocketServer.py", line 649, in __init__
    self.handle()
    self.handle()
  File "/usr/lib64/python2.7/wsgiref/simple_server.py", line 117, in handle
  File "/usr/lib64/python2.7/wsgiref/simple_server.py", line 117, in handle
    if not self.parse_request(): # An error code has been sent, just exit
    if not self.parse_request(): # An error code has been sent, just exit
  File "/usr/lib64/python2.7/BaseHTTPServer.py", line 286, in parse_request
  File "/usr/lib64/python2.7/BaseHTTPServer.py", line 281, in parse_request
    self.send_error(400, "Bad request syntax (%r)" % requestline)
    "Bad HTTP/0.9 request type (%r)" % command)
  File "/usr/lib64/python2.7/BaseHTTPServer.py", line 368, in send_error
  File "/usr/lib64/python2.7/BaseHTTPServer.py", line 368, in send_error
    self.send_response(code, message)
    self.send_response(code, message)
  File "/usr/lib64/python2.7/BaseHTTPServer.py", line 385, in send_response
  File "/usr/lib64/python2.7/BaseHTTPServer.py", line 385, in send_response
    self.log_request(code)
    self.log_request(code)
  File "/usr/lib64/python2.7/BaseHTTPServer.py", line 422, in log_request
  File "/usr/lib64/python2.7/BaseHTTPServer.py", line 422, in log_request
    self.requestline, str(code), str(size))
    self.requestline, str(code), str(size))
  File "/usr/lib/python2.7/site-packages/django/core/servers/basehttp.py", line 162, in log_message
  File "/usr/lib/python2.7/site-packages/django/core/servers/basehttp.py", line 162, in log_message
    msg = "[%s] %s\n" % (self.log_date_time_string(), format % args)
    msg = "[%s] %s\n" % (self.log_date_time_string(), format % args)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xce in position 5: ordinal not in range(128)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xce in position 5: ordinal not in range(128)
/usr/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

Internal Server Error: /
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/django/core/handlers/base.py", line 103, in get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/usr/lib/python2.7/site-packages/django/core/urlresolvers.py", line 321, in resolve
    sub_match = pattern.resolve(new_path)
  File "/usr/lib/python2.7/site-packages/django/core/urlresolvers.py", line 223, in resolve
    return ResolverMatch(self.callback, args, kwargs, self.name)
  File "/usr/lib/python2.7/site-packages/django/core/urlresolvers.py", line 230, in callback
    self._callback = get_callable(self._callback_str)
  File "/usr/lib/python2.7/site-packages/django/utils/functional.py", line 29, in wrapper
    result = func(*args)
  File "/usr/lib/python2.7/site-packages/django/core/urlresolvers.py", line 97, in get_callable
    mod = import_module(mod_name)
  File "/usr/lib/python2.7/site-packages/django/utils/importlib.py", line 35, in import_module
    __import__(name)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 10, in <module>
    from bs4 import BeautifulSoup
ImportError: No module named bs4
[01/Feb/2018 13:20:11] "GET / HTTP/1.1" 500 87821
Internal Server Error: /
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/django/core/handlers/base.py", line 103, in get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/usr/lib/python2.7/site-packages/django/core/urlresolvers.py", line 321, in resolve
    sub_match = pattern.resolve(new_path)
  File "/usr/lib/python2.7/site-packages/django/core/urlresolvers.py", line 223, in resolve
    return ResolverMatch(self.callback, args, kwargs, self.name)
  File "/usr/lib/python2.7/site-packages/django/core/urlresolvers.py", line 230, in callback
    self._callback = get_callable(self._callback_str)
  File "/usr/lib/python2.7/site-packages/django/utils/functional.py", line 29, in wrapper
    result = func(*args)
  File "/usr/lib/python2.7/site-packages/django/core/urlresolvers.py", line 97, in get_callable
    mod = import_module(mod_name)
  File "/usr/lib/python2.7/site-packages/django/utils/importlib.py", line 35, in import_module
    __import__(name)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 11, in <module>
    from elasticsearch import Elasticsearch, helpers
ImportError: No module named elasticsearch
[01/Feb/2018 13:20:58] "GET / HTTP/1.1" 500 88127
[01/Feb/2018 13:31:57] "GET / HTTP/1.1" 200 1848
[01/Feb/2018 13:31:57] "GET /static/rsc/chem.css HTTP/1.1" 200 3082
[01/Feb/2018 13:31:57] "GET /static/ist441list.jpg HTTP/1.1" 404 1660
[01/Feb/2018 13:32:16] "POST / HTTP/1.1" 200 2814
[01/Feb/2018 13:32:16] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
[01/Feb/2018 13:32:16] "GET /static/rsc/reset.css HTTP/1.1" 200 939
[01/Feb/2018 13:32:16] "GET /static/rsc/result.css HTTP/1.1" 200 1513
[01/Feb/2018 13:32:16] "GET /static/rsc/tc-mod.js HTTP/1.1" 200 3642
[01/Feb/2018 13:32:16] "GET /static/rsc/view.css HTTP/1.1" 200 2715
[01/Feb/2018 13:32:16] "GET /static/rsc/fusion-logo-resized-small.png HTTP/1.1" 200 1219
[01/Feb/2018 13:32:25] "POST / HTTP/1.1" 200 2814
[01/Feb/2018 13:32:25] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
[01/Feb/2018 13:32:30] "GET / HTTP/1.1" 200 1848
/usr/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

[19/Mar/2018 11:15:38] "GET / HTTP/1.1" 200 1848
[19/Mar/2018 11:15:38] "GET /static/rsc/chem.css HTTP/1.1" 200 3082
[19/Mar/2018 11:15:38] "GET /static/ist441list.jpg HTTP/1.1" 404 1660
[19/Mar/2018 11:15:39] "GET /static/ist441list.jpg HTTP/1.1" 404 1660
[19/Mar/2018 11:17:38] "GET / HTTP/1.1" 200 1848
[19/Mar/2018 11:17:38] "GET /static/rsc/chem.css HTTP/1.1" 200 3082
[19/Mar/2018 11:17:38] "GET /static/ist441list.jpg HTTP/1.1" 404 1660
[19/Mar/2018 11:17:45] "POST / HTTP/1.1" 200 2814
[19/Mar/2018 11:17:45] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
[19/Mar/2018 11:17:45] "GET /static/rsc/reset.css HTTP/1.1" 200 939
[19/Mar/2018 11:17:45] "GET /static/rsc/tc-mod.js HTTP/1.1" 200 3642
[19/Mar/2018 11:17:45] "GET /static/rsc/result.css HTTP/1.1" 200 1513
[19/Mar/2018 11:17:45] "GET /static/rsc/view.css HTTP/1.1" 200 2715
[19/Mar/2018 11:17:45] "GET /static/rsc/fusion-logo-resized-small.png HTTP/1.1" 200 1219
[19/Mar/2018 11:18:26] "POST / HTTP/1.1" 200 2814
[19/Mar/2018 11:18:26] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
[19/Mar/2018 11:18:39] "POST / HTTP/1.1" 200 2814
[19/Mar/2018 11:18:39] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
[19/Mar/2018 11:18:40] "POST / HTTP/1.1" 200 1984
[19/Mar/2018 11:18:40] "GET /static/ist441list.jpg HTTP/1.1" 404 1660
[19/Mar/2018 11:18:46] "POST / HTTP/1.1" 200 2814
[19/Mar/2018 11:18:46] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
[19/Mar/2018 11:18:51] "POST / HTTP/1.1" 200 2814
[19/Mar/2018 11:18:51] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
[19/Mar/2018 11:19:16] "GET / HTTP/1.1" 200 1848
[19/Mar/2018 11:19:16] "GET /static/ist441list.jpg HTTP/1.1" 404 1660
[19/Mar/2018 14:55:12] "GET / HTTP/1.1" 200 1848
[19/Mar/2018 14:55:12] "GET /static/rsc/chem.css HTTP/1.1" 200 3082
[19/Mar/2018 14:55:12] "GET /static/ist441list.jpg HTTP/1.1" 404 1660
[19/Mar/2018 14:55:12] "GET /static/ist441list.jpg HTTP/1.1" 404 1660
Validating models...

0 errors found
March 19, 2018 - 11:15:04
Django version 1.5.1, using settings 'interface.settings'
Development server is running at http://ist441giles.ist.psu.edu:5001/
Quit the server with CONTROL-C.
search done
search done
search done
search done
search done
/usr/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

Internal Server Error: /
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 37, in home
    return search(request,q,start)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 73, in search
    resultid= result['_source']['_id']
KeyError: '_id'
[19/Mar/2018 15:00:13] "POST / HTTP/1.1" 500 396795
Validating models...

0 errors found
March 19, 2018 - 14:57:30
Django version 1.5.1, using settings 'interface.settings'
Development server is running at http://ist441giles.ist.psu.edu:5001/
Quit the server with CONTROL-C.
search done
/usr/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

Internal Server Error: /
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 37, in home
    return search(request,q,start)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 74, in search
    f = SearchResult(resultid) #calling the object class that is defined inside models.py
TypeError: __init__() takes exactly 6 arguments (2 given)
[19/Mar/2018 15:02:13] "POST / HTTP/1.1" 500 397399
Validating models...

0 errors found
March 19, 2018 - 15:02:09
Django version 1.5.1, using settings 'interface.settings'
Development server is running at http://ist441giles.ist.psu.edu:5001/
Quit the server with CONTROL-C.
search done
Validating models...

0 errors found
March 19, 2018 - 15:02:34
Django version 1.5.1, using settings 'interface.settings'
Development server is running at http://ist441giles.ist.psu.edu:5001/
Quit the server with CONTROL-C.
/usr/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

Internal Server Error: /
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 37, in home
    return search(request,q,start)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 93, in search
    SearchResults.append(f)
NameError: global name 'SearchResults' is not defined
[19/Mar/2018 15:04:18] "POST / HTTP/1.1" 500 398167
Validating models...

0 errors found
March 19, 2018 - 15:04:13
Django version 1.5.1, using settings 'interface.settings'
Development server is running at http://ist441giles.ist.psu.edu:5001/
Quit the server with CONTROL-C.
search done
/usr/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

Internal Server Error: /
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 37, in home
    return search(request,q,start)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 85, in search
    f.description = result['_source']['meta']['raw']['description']
KeyError: 'description'
[19/Mar/2018 15:05:22] "POST / HTTP/1.1" 500 394231
Validating models...

0 errors found
March 19, 2018 - 15:04:42
Django version 1.5.1, using settings 'interface.settings'
Development server is running at http://ist441giles.ist.psu.edu:5001/
Quit the server with CONTROL-C.
search done
/usr/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

[19/Mar/2018 15:08:17] "POST / HTTP/1.1" 200 7200
[19/Mar/2018 15:08:17] "GET /static/rsc/result.css HTTP/1.1" 200 1513
[19/Mar/2018 15:08:17] "GET /static/rsc/reset.css HTTP/1.1" 200 939
[19/Mar/2018 15:08:17] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
[19/Mar/2018 15:08:17] "GET /static/rsc/chem.css HTTP/1.1" 304 0
[19/Mar/2018 15:08:17] "GET /static/rsc/tc-mod.js HTTP/1.1" 200 3642
[19/Mar/2018 15:08:17] "GET /static/rsc/view.css HTTP/1.1" 200 2715
[19/Mar/2018 15:08:17] "GET /static/rsc/fusion-logo-resized-small.png HTTP/1.1" 200 1219
[19/Mar/2018 15:08:38] "GET /?q=Citeseer&start=6 HTTP/1.1" 200 7204
[19/Mar/2018 15:08:38] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
[19/Mar/2018 15:08:42] "GET /?q=Citeseer&start=6 HTTP/1.1" 200 7204
[19/Mar/2018 15:08:42] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
Validating models...

0 errors found
March 19, 2018 - 15:08:13
Django version 1.5.1, using settings 'interface.settings'
Development server is running at http://ist441giles.ist.psu.edu:5001/
Quit the server with CONTROL-C.
search done
search done
search done
/usr/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

Internal Server Error: /
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 37, in home
    return search(request,q,start)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 85, in search
    f.description = result['_source']['meta']['description']
KeyError: 'description'
[19/Mar/2018 15:10:02] "POST / HTTP/1.1" 500 397996
Validating models...

0 errors found
March 19, 2018 - 15:09:59
Django version 1.5.1, using settings 'interface.settings'
Development server is running at http://ist441giles.ist.psu.edu:5001/
Quit the server with CONTROL-C.
search done
/usr/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

Internal Server Error: /
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 37, in home
    return search(request,q,start)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 85, in search
    f.description = result['_source']['meta']['raw']['description.keyword']
KeyError: 'description.keyword'
[19/Mar/2018 15:11:14] "POST / HTTP/1.1" 500 398074
Validating models...

0 errors found
March 19, 2018 - 15:11:09
Django version 1.5.1, using settings 'interface.settings'
Development server is running at http://ist441giles.ist.psu.edu:5001/
Quit the server with CONTROL-C.
search done
/usr/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

[19/Mar/2018 15:11:40] "POST / HTTP/1.1" 200 8976
[19/Mar/2018 15:11:40] "GET /static/rsc/chem.css HTTP/1.1" 304 0
[19/Mar/2018 15:11:41] "GET /static/rsc/reset.css HTTP/1.1" 304 0
[19/Mar/2018 15:11:41] "GET /static/rsc/result.css HTTP/1.1" 304 0
[19/Mar/2018 15:11:41] "GET /static/rsc/view.css HTTP/1.1" 304 0
[19/Mar/2018 15:11:41] "GET /static/rsc/tc-mod.js HTTP/1.1" 304 0
[19/Mar/2018 15:11:41] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
[19/Mar/2018 15:11:41] "GET /static/rsc/fusion-logo-resized-small.png HTTP/1.1" 304 0
Validating models...

0 errors found
March 19, 2018 - 15:11:38
Django version 1.5.1, using settings 'interface.settings'
Development server is running at http://ist441giles.ist.psu.edu:5001/
Quit the server with CONTROL-C.
search done
/usr/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

Internal Server Error: /
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 37, in home
    return search(request,q,start)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 85, in search
    f.description = result['_source']['meta']['raw'][u'description']
KeyError: u'description'
[19/Mar/2018 15:12:35] "POST / HTTP/1.1" 500 394237
Internal Server Error: /
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 37, in home
    return search(request,q,start)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 85, in search
    f.description = result['_source']['meta']['raw'][u'description']
KeyError: u'description'
[19/Mar/2018 15:12:42] "POST / HTTP/1.1" 500 393943
Validating models...

0 errors found
March 19, 2018 - 15:12:31
Django version 1.5.1, using settings 'interface.settings'
Development server is running at http://ist441giles.ist.psu.edu:5001/
Quit the server with CONTROL-C.
search done
search done
/usr/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

[19/Mar/2018 15:14:18] "POST / HTTP/1.1" 200 7224
[19/Mar/2018 15:14:18] "GET /static/rsc/chem.css HTTP/1.1" 304 0
[19/Mar/2018 15:14:18] "GET /static/rsc/reset.css HTTP/1.1" 304 0
[19/Mar/2018 15:14:18] "GET /static/rsc/result.css HTTP/1.1" 304 0
[19/Mar/2018 15:14:18] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
[19/Mar/2018 15:14:18] "GET /static/rsc/tc-mod.js HTTP/1.1" 304 0
[19/Mar/2018 15:14:18] "GET /static/rsc/view.css HTTP/1.1" 304 0
[19/Mar/2018 15:14:18] "GET /static/rsc/fusion-logo-resized-small.png HTTP/1.1" 304 0
Validating models...

0 errors found
March 19, 2018 - 15:14:13
Django version 1.5.1, using settings 'interface.settings'
Development server is running at http://ist441giles.ist.psu.edu:5001/
Quit the server with CONTROL-C.
search done
Unhandled exception in thread started by <bound method Command.inner_run of <django.contrib.staticfiles.management.commands.runserver.Command object at 0x2754890>>/usr/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

Internal Server Error: /
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 37, in home
    return search(request,q,start)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 85, in search
    f.description = result['_source']['meta']['raw']['description']
KeyError: 'description'
[19/Mar/2018 15:15:07] "POST / HTTP/1.1" 500 394228
Validating models...

0 errors found
March 19, 2018 - 15:15:02
Django version 1.5.1, using settings 'interface.settings'
Development server is running at http://ist441giles.ist.psu.edu:5001/
Quit the server with CONTROL-C.
search done
/usr/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

Internal Server Error: /
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 37, in home
    return search(request,q,start)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 85, in search
    f.description = result['_source']['meta']['raw'].encoding('utf-8') #['description']
AttributeError: 'dict' object has no attribute 'encoding'
[19/Mar/2018 15:18:46] "POST / HTTP/1.1" 500 398274
Validating models...

0 errors found
March 19, 2018 - 15:18:42
Django version 1.5.1, using settings 'interface.settings'
Development server is running at http://ist441giles.ist.psu.edu:5001/
Quit the server with CONTROL-C.
search done
/usr/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

Internal Server Error: /
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 37, in home
    return search(request,q,start)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 85, in search
    f.description = result['_source']['meta']['raw'].encode('utf-8') #['description']
AttributeError: 'dict' object has no attribute 'encode'
[19/Mar/2018 15:19:07] "POST / HTTP/1.1" 500 398262
Validating models...

0 errors found
March 19, 2018 - 15:19:04
Django version 1.5.1, using settings 'interface.settings'
Development server is running at http://ist441giles.ist.psu.edu:5001/
Quit the server with CONTROL-C.
search done
/usr/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

[19/Mar/2018 15:19:36] "POST / HTTP/1.1" 200 8976
[19/Mar/2018 15:19:36] "GET /static/rsc/result.css HTTP/1.1" 304 0
[19/Mar/2018 15:19:36] "GET /static/rsc/chem.css HTTP/1.1" 304 0
[19/Mar/2018 15:19:36] "GET /static/rsc/reset.css HTTP/1.1" 304 0
[19/Mar/2018 15:19:36] "GET /static/rsc/view.css HTTP/1.1" 304 0
[19/Mar/2018 15:19:36] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
[19/Mar/2018 15:19:36] "GET /static/rsc/tc-mod.js HTTP/1.1" 304 0
[19/Mar/2018 15:19:36] "GET /static/rsc/fusion-logo-resized-small.png HTTP/1.1" 304 0
Validating models...

0 errors found
March 19, 2018 - 15:19:31
Django version 1.5.1, using settings 'interface.settings'
Development server is running at http://ist441giles.ist.psu.edu:5001/
Quit the server with CONTROL-C.
search done
/usr/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

[19/Mar/2018 15:22:52] "POST / HTTP/1.1" 200 8976
[19/Mar/2018 15:22:52] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
[19/Mar/2018 15:22:52] "GET /static/rsc/chem.css HTTP/1.1" 304 0
[19/Mar/2018 15:22:52] "GET /static/rsc/result.css HTTP/1.1" 304 0
[19/Mar/2018 15:22:52] "GET /static/rsc/view.css HTTP/1.1" 304 0
[19/Mar/2018 15:22:52] "GET /static/rsc/reset.css HTTP/1.1" 304 0
[19/Mar/2018 15:22:52] "GET /static/rsc/tc-mod.js HTTP/1.1" 304 0
[19/Mar/2018 15:22:53] "GET /static/rsc/fusion-logo-resized-small.png HTTP/1.1" 304 0
[19/Mar/2018 15:22:54] "GET /?q=Citeseer&start=6 HTTP/1.1" 200 2814
[19/Mar/2018 15:22:55] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
[19/Mar/2018 15:23:06] "POST / HTTP/1.1" 200 14817
[19/Mar/2018 15:23:06] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
[19/Mar/2018 15:23:08] "GET /?q=project&start=10 HTTP/1.1" 200 4368
[19/Mar/2018 15:23:08] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
Validating models...

0 errors found
March 19, 2018 - 15:22:42
Django version 1.5.1, using settings 'interface.settings'
Development server is running at http://ist441giles.ist.psu.edu:5001/
Quit the server with CONTROL-C.
search done
search done
search done
search done
/usr/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

[19/Mar/2018 15:26:57] "GET / HTTP/1.1" 200 1848
[19/Mar/2018 15:26:57] "GET /static/rsc/ist441list.jpg HTTP/1.1" 404 1672
[19/Mar/2018 15:27:57] "POST / HTTP/1.1" 200 14817
[19/Mar/2018 15:27:57] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
[19/Mar/2018 15:27:59] "GET /?q=project&start=10 HTTP/1.1" 200 4368
[19/Mar/2018 15:27:59] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
[19/Mar/2018 15:28:07] "GET /?q=project&start=10 HTTP/1.1" 200 4368
[19/Mar/2018 15:28:07] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
[19/Mar/2018 15:29:46] "GET /?q=project&start=10 HTTP/1.1" 200 4393
[19/Mar/2018 15:29:46] "GET /static/rsc/chem.css HTTP/1.1" 304 0
[19/Mar/2018 15:29:46] "GET /static/rsc/reset.css HTTP/1.1" 304 0
[19/Mar/2018 15:29:46] "GET /static/rsc/result.css HTTP/1.1" 304 0
[19/Mar/2018 15:29:46] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
[19/Mar/2018 15:29:46] "GET /static/rsc/view.css HTTP/1.1" 304 0
[19/Mar/2018 15:29:46] "GET /static/rsc/tc-mod.js HTTP/1.1" 304 0
[19/Mar/2018 15:29:46] "GET /static/rsc/fusion-logo-resized-small.png HTTP/1.1" 304 0
Validating models...

0 errors found
March 19, 2018 - 15:23:45
Django version 1.5.1, using settings 'interface.settings'
Development server is running at http://ist441giles.ist.psu.edu:5001/
Quit the server with CONTROL-C.
search done
search done
search done
search done
/usr/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

[19/Mar/2018 15:31:32] "GET /?q=project&start=10 HTTP/1.1" 200 3837
[19/Mar/2018 15:31:32] "GET /static/rsc/chem.css HTTP/1.1" 304 0
[19/Mar/2018 15:31:32] "GET /static/rsc/reset.css HTTP/1.1" 304 0
[19/Mar/2018 15:31:32] "GET /static/rsc/view.css HTTP/1.1" 304 0
[19/Mar/2018 15:31:32] "GET /static/rsc/result.css HTTP/1.1" 304 0
[19/Mar/2018 15:31:32] "GET /static/rsc/tc-mod.js HTTP/1.1" 304 0
[19/Mar/2018 15:31:32] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
[19/Mar/2018 15:31:32] "GET /static/rsc/fusion-logo-resized-small.png HTTP/1.1" 304 0
Internal Server Error: /
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 37, in home
    return search(request,q,start)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 87, in search
    f.description = str(result['_source']['meta']['raw']['description'])
KeyError: 'description'
[19/Mar/2018 15:31:40] "POST / HTTP/1.1" 500 266058
Internal Server Error: /
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 37, in home
    return search(request,q,start)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 87, in search
    f.description = str(result['_source']['meta']['raw']['description'])
KeyError: 'description'
[19/Mar/2018 15:31:44] "POST / HTTP/1.1" 500 265764
Validating models...

0 errors found
March 19, 2018 - 15:31:30
Django version 1.5.1, using settings 'interface.settings'
Development server is running at http://ist441giles.ist.psu.edu:5001/
Quit the server with CONTROL-C.
search done
search done
search done
/usr/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

[19/Mar/2018 15:34:21] "POST / HTTP/1.1" 200 14842
[19/Mar/2018 15:34:21] "GET /static/rsc/result.css HTTP/1.1" 304 0
[19/Mar/2018 15:34:21] "GET /static/rsc/reset.css HTTP/1.1" 304 0
[19/Mar/2018 15:34:21] "GET /static/rsc/chem.css HTTP/1.1" 304 0
[19/Mar/2018 15:34:21] "GET /static/rsc/view.css HTTP/1.1" 304 0
[19/Mar/2018 15:34:21] "GET /static/rsc/tc-mod.js HTTP/1.1" 304 0
[19/Mar/2018 15:34:21] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
[19/Mar/2018 15:34:21] "GET /static/rsc/fusion-logo-resized-small.png HTTP/1.1" 304 0
[19/Mar/2018 15:34:23] "GET /?q=project&start=10 HTTP/1.1" 200 4393
[19/Mar/2018 15:34:23] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
Validating models...

0 errors found
March 19, 2018 - 15:34:18
Django version 1.5.1, using settings 'interface.settings'
Development server is running at http://ist441giles.ist.psu.edu:5001/
Quit the server with CONTROL-C.
search done
search done
/usr/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

[19/Mar/2018 15:34:45] "GET /?q=project&start=10 HTTP/1.1" 200 3837
[19/Mar/2018 15:34:45] "GET /static/rsc/chem.css HTTP/1.1" 304 0
[19/Mar/2018 15:34:45] "GET /static/rsc/reset.css HTTP/1.1" 304 0
[19/Mar/2018 15:34:45] "GET /static/rsc/result.css HTTP/1.1" 304 0
[19/Mar/2018 15:34:45] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
[19/Mar/2018 15:34:45] "GET /static/rsc/tc-mod.js HTTP/1.1" 304 0
[19/Mar/2018 15:34:45] "GET /static/rsc/view.css HTTP/1.1" 304 0
[19/Mar/2018 15:34:45] "GET /static/rsc/fusion-logo-resized-small.png HTTP/1.1" 304 0
Validating models...

0 errors found
March 19, 2018 - 15:34:42
Django version 1.5.1, using settings 'interface.settings'
Development server is running at http://ist441giles.ist.psu.edu:5001/
Quit the server with CONTROL-C.
search done
/usr/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

Internal Server Error: /
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 52, in home
    return search(request,query,start)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 88, in search
    f.description =result['_source']['highlight']['content']
KeyError: 'highlight'
[19/Mar/2018 15:41:37] "GET /?q=project&start=10 HTTP/1.1" 500 91469
[19/Mar/2018 15:52:27] "GET / HTTP/1.1" 200 1848
[19/Mar/2018 15:52:27] "GET /static/rsc/chem.css HTTP/1.1" 200 3082
[19/Mar/2018 15:52:27] "GET /static/rsc/ist441list.jpg HTTP/1.1" 404 1672
Internal Server Error: /
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 37, in home
    return search(request,q,start)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 88, in search
    f.description =result['_source']['highlight']['content']
KeyError: 'highlight'
[19/Mar/2018 15:52:39] "POST / HTTP/1.1" 500 397693
[19/Mar/2018 15:54:08] "GET / HTTP/1.1" 200 1848
Validating models...

0 errors found
March 19, 2018 - 15:41:17
Django version 1.5.1, using settings 'interface.settings'
Development server is running at http://ist441giles.ist.psu.edu:5001/
Quit the server with CONTROL-C.
search done
search done
/usr/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

[19/Mar/2018 15:56:57] "GET / HTTP/1.1" 200 1848
[19/Mar/2018 15:56:57] "GET /static/rsc/ist441list.jpg HTTP/1.1" 404 1672
Internal Server Error: /
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 37, in home
    return search(request,q,start)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 88, in search
    f.description =result['_source']['highlight']['content']
KeyError: 'highlight'
[19/Mar/2018 15:57:00] "POST / HTTP/1.1" 500 406444
Validating models...

0 errors found
March 19, 2018 - 15:56:55
Django version 1.5.1, using settings 'interface.settings'
Development server is running at http://ist441giles.ist.psu.edu:5001/
Quit the server with CONTROL-C.
search done
/usr/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

[19/Mar/2018 15:59:32] "POST / HTTP/1.1" 200 9565
[19/Mar/2018 15:59:32] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
[19/Mar/2018 15:59:32] "GET /static/rsc/reset.css HTTP/1.1" 200 939
[19/Mar/2018 15:59:32] "GET /static/rsc/result.css HTTP/1.1" 200 1513
[19/Mar/2018 15:59:32] "GET /static/rsc/view.css HTTP/1.1" 200 2715
[19/Mar/2018 15:59:32] "GET /static/rsc/tc-mod.js HTTP/1.1" 200 3642
[19/Mar/2018 15:59:32] "GET /static/rsc/fusion-logo-resized-small.png HTTP/1.1" 200 1219
Validating models...

0 errors found
March 19, 2018 - 15:59:29
Django version 1.5.1, using settings 'interface.settings'
Development server is running at http://ist441giles.ist.psu.edu:5001/
Quit the server with CONTROL-C.
search done
/usr/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

Internal Server Error: /
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 37, in home
    return search(request,q,start)
  File "/data/ist441/team1/search/interface/rsc/views.py", line 90, in search
    f.description = f.description.encode("utf-8")
AttributeError: 'list' object has no attribute 'encode'
[19/Mar/2018 16:01:22] "POST / HTTP/1.1" 500 406602
Validating models...

0 errors found
March 19, 2018 - 16:01:19
Django version 1.5.1, using settings 'interface.settings'
Development server is running at http://ist441giles.ist.psu.edu:5001/
Quit the server with CONTROL-C.
search done
/usr/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

[19/Mar/2018 16:02:07] "POST / HTTP/1.1" 200 9405
[19/Mar/2018 16:02:07] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
Validating models...

0 errors found
March 19, 2018 - 16:02:03
Django version 1.5.1, using settings 'interface.settings'
Development server is running at http://ist441giles.ist.psu.edu:5001/
Quit the server with CONTROL-C.
search done
/usr/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

[19/Mar/2018 16:05:18] "POST / HTTP/1.1" 200 9447
[19/Mar/2018 16:05:18] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
/usr/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

[19/Mar/2018 16:08:25] "GET / HTTP/1.1" 200 1848
[19/Mar/2018 16:08:25] "GET /static/rsc/ist441list.jpg HTTP/1.1" 404 1672
[19/Mar/2018 16:08:25] "GET /static/rsc/chem.css HTTP/1.1" 200 3082
[19/Mar/2018 16:08:29] "POST / HTTP/1.1" 200 2814
[19/Mar/2018 16:08:29] "GET /static/rsc/reset.css HTTP/1.1" 200 939
[19/Mar/2018 16:08:29] "GET /static/rsc/chem.css HTTP/1.1" 304 0
[19/Mar/2018 16:08:29] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
[19/Mar/2018 16:08:29] "GET /static/rsc/result.css HTTP/1.1" 200 1513
[19/Mar/2018 16:08:29] "GET /static/rsc/tc-mod.js HTTP/1.1" 200 3642
[19/Mar/2018 16:08:29] "GET /static/rsc/view.css HTTP/1.1" 200 2715
[19/Mar/2018 16:08:29] "GET /static/rsc/fusion-logo-resized-small.png HTTP/1.1" 200 1219
/usr/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

[19/Mar/2018 16:09:50] "POST / HTTP/1.1" 200 9447
[19/Mar/2018 16:09:50] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
[19/Mar/2018 16:09:50] "GET /static/rsc/reset.css HTTP/1.1" 304 0
[19/Mar/2018 16:09:50] "GET /static/rsc/tc-mod.js HTTP/1.1" 304 0
[19/Mar/2018 16:09:50] "GET /static/rsc/view.css HTTP/1.1" 304 0
[19/Mar/2018 16:09:50] "GET /static/rsc/result.css HTTP/1.1" 304 0
[19/Mar/2018 16:09:50] "GET /static/rsc/chem.css HTTP/1.1" 304 0
[19/Mar/2018 16:09:50] "GET /static/rsc/fusion-logo-resized-small.png HTTP/1.1" 304 0
[19/Mar/2018 16:12:36] "GET / HTTP/1.1" 200 1848
/usr/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

[21/Mar/2018 12:40:55] "GET / HTTP/1.1" 200 1848
[21/Mar/2018 12:40:55] "GET /static/rsc/chem.css HTTP/1.1" 200 3082
[21/Mar/2018 12:40:55] "GET /static/rsc/ist441list.jpg HTTP/1.1" 404 1672
[21/Mar/2018 12:40:59] "POST / HTTP/1.1" 200 2814
[21/Mar/2018 12:40:59] "GET /static/chemseer/css/expert.css HTTP/1.1" 404 1687
[21/Mar/2018 12:40:59] "GET /static/rsc/reset.css HTTP/1.1" 200 939
[21/Mar/2018 12:40:59] "GET /static/rsc/result.css HTTP/1.1" 200 1513
[21/Mar/2018 12:40:59] "GET /static/rsc/tc-mod.js HTTP/1.1" 200 3642
[21/Mar/2018 12:40:59] "GET /static/rsc/view.css HTTP/1.1" 200 2715
[21/Mar/2018 12:40:59] "GET /static/rsc/fusion-logo-resized-small.png HTTP/1.1" 200 1219
Traceback (most recent call last):
  File "/usr/lib64/python2.7/SocketServer.py", line 295, in _handle_request_noblock
    self.process_request(request, client_address)
  File "/usr/lib64/python2.7/SocketServer.py", line 604, in process_request
    t.start()
  File "/usr/lib64/python2.7/threading.py", line 747, in start
    _start_new_thread(self.__bootstrap, ())
error: can't start new thread
Traceback (most recent call last):
  File "/usr/lib64/python2.7/SocketServer.py", line 295, in _handle_request_noblock
    self.process_request(request, client_address)
  File "/usr/lib64/python2.7/SocketServer.py", line 604, in process_request
    t.start()
  File "/usr/lib64/python2.7/threading.py", line 747, in start
    _start_new_thread(self.__bootstrap, ())
error: can't start new thread
Traceback (most recent call last):
  File "/usr/lib64/python2.7/SocketServer.py", line 295, in _handle_request_noblock
    self.process_request(request, client_address)
  File "/usr/lib64/python2.7/SocketServer.py", line 604, in process_request
    t.start()
  File "/usr/lib64/python2.7/threading.py", line 747, in start
    _start_new_thread(self.__bootstrap, ())
error: can't start new thread
